x-pack/plugins/lens/public/trigger_actions/open_lens_config/edit_action_helpers.ts

```diff
@@ -24,6 +24,7 @@ interface Context {
 }
 
 export async function isEditActionCompatible(embeddable: IEmbeddable) {
+  if (!embeddable?.getInput) return false;
```

Comment:
If this isn't just temporary, it would be nice to see a comment. If you're going to change this anyway, disregard.

Reply:
This is just temporary until https://github.com/elastic/kibana/issues/175138 is completed. It's there just to avoid a runtime error when a React Embeddable is on the Dashboard.